features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,20 @@
-# put step definitions in this file
\ No newline at end of file
+When(/^I visit the index site$/) do
+   visit '/'
+end
+
+Then(/^I should see "([^"]*)"$/) do |arg1|
+   expect(page).to have_content arg1
+end
+
+When(/^I visit the name site$/) do
+   visit '/name/'
+end
+
+
+When(/^I visit the name page for "([^"]*)"$/) do |arg1|
+  visit '/name/:Casper'
+end
```

Comment:
Here you are not using the variable `arg1`. Consider this line: `visit "/name/#{arg1}"`